src/field_theory/separable.lean

```diff
@@ -482,3 +508,26 @@ end
 the minimal polynomial of every `x : K` is separable. -/
 @[class] def is_separable (F K : Sort*) [field F] [field K] [algebra F K] : Prop :=
 ∀ x : K, ∃ H : is_integral F x, (minimal_polynomial H).separable
+
+section is_separable_tower
+variables (F K E : Type*) [field F] [field K] [field E] [algebra F K] [algebra F E] [algebra K E]
```

Comment:
You should be able to leave `F` and `E` implicit in this section, because they can be inferred from `h : is_separable F E`.